tests/examples/awful/mouse/coords.lua

```diff
@@ -0,0 +1,13 @@
+screen[1]._resize {x = 175, width = 128, height = 96} --DOC_HIDE
+mouse.coords {x=175+60,y=60} --DOC_HIDE
+
+ -- Get the position
+print(mouse.coords().x)
+ 
```

Comment:
That's... bad. Eventually this trailing whitespace will be removed by someone.
However, feel free to try to leave it in for now.